### DISCUSSION TITLE
HowTo

### DISCUSSION THREAD

**Discussion by ctv99** [Sanjeet990/Astroluma#17] [General]
Cant make Proxmox Work Across my VPN? does anyone have a Setup Guide yet?

**Comment by Sanjeet990** [comment]
Not yet but would love to help you out in this. Let's catch up some time may be?

**Comment by ctv99** [comment]
Would be Great to have Proxmox Setting just allow a list of IPS to Connect Too or Just manually Add Devices with Check boxes for Account Credentials.

[A person with a beard  Description automatically generated]
[A green and white logo  Description automatically generated]


Charles Stevens
Operations Manager
[Title: LinkedIn - Description: LinkedIn icon]<https://www.linkedin.com/in/charles-stevens-632419a1>[Title: Facebook - Description: Facebook icon]<https://www.fb.com/scwks>

20 S. Main St., Hutchinson, KS 67501
Phone: 620.860.0996 | 620.728.0555x3
Open 9am to 6pm M-F

www.scw.biz<https://www.scw.biz/> | Book a ***@***.***/bookings/> | Current Inventory<https://www.scw.biz/inventory/>
